components/Modules/parts/ModuleIconEditor.tsx

```
@@ -0,0 +1,86 @@
+import { Button } from "@/components/ui/button";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+import { Module } from "../module-detail-page";
```

ESLint

[error] 'Module' is defined but never used.

(@typescript-eslint/no-unused-vars)